lib/cli/src/frameworks/lit/js/Button.stories.js

```diff
@@ -0,0 +1,51 @@
+import './Button';
+import { html } from 'lit';
+
+export default {
+  title: 'Example/Button',
+  argTypes: {
+    backgroundColor: { control: 'color' },
+    onClick: { action: 'onClick' },
+  },
+  parameters: {
+    actions: {
+      handles: ['click', 'sb-button:click'],
+    },
+  },
+};
+
+const Template = ({ primary, backgroundColor, size, label, onClick }) =>
+  html`<sb-button
+    ?primary="${primary}"
+    .size="${size}"
+    .label="${label}"
+    .backgroundColor="${backgroundColor}"
+    @sb-button:click="${onClick}"
+  ></sb-button>`;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  primary: true,
+  label: 'Button',
+  onClick: (e) => console.log(`Clicked`, e),
```

Comment:
I'm really curious, will this log + be shown in the actions panel? I thought it would override actions and just log 🤔

Reply:
Yeah AFAIK user code takes precedence over the automated code (at least that was the design). So actions would get displaced by the logging.

I never considered the other direction, which is also interesting: user provides simple code that works "without magic" and then the magic comes along and wires it into actions. But would be a major change and inconsistent with everything else in Storybook.